database/comments.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+const db = require('./index.js');
+mongoose.Promise = global.Promise;
+
+const commentSection = new mongoose.Schema({
+  id: { type: Number, unique: true, required: true },
+  recipeName: String,
+  fullName: String,
+  comments: String,
```

Comment:
I'm not sure I understand your schema's logic. It looks like you are creating a new document for each comment, which is linked to a recipe through a recipe name field. This is going to make things pretty inefficient to find the comments associated with a recipe. Lets talk about this in person.